cartographer/sensor/test_helpers.h

```diff
@@ -29,6 +36,47 @@ MATCHER_P(Near, point, std::string(negation ? "Doesn't" : "Does") + " match.") {
   return arg.isApprox(point, 0.001f);
 }
 
+namespace test {
+
+typedef std::tuple<int, std::string, common::Time> CollatorOutput;
```

Comment:
can you add /* trajectory_id */ and such to make clear what those ints and strings are?

Reply:
Done